core/src/main/java/org/apache/iceberg/Partitioning.java

```diff
@@ -222,27 +224,41 @@ public static StructType partitionType(Table table) {
 
       for (PartitionField field : spec.fields()) {
         int fieldId = field.fieldId();
+        NestedField structField = spec.partitionType().field(fieldId);
+
```

Comment:
Nit: no need for a newline between assignment statements.

Reply:
removed